include/measurement_kit/neubot/dash.hpp

```diff
@@ -0,0 +1,26 @@
+// Part of measurement-kit <https://measurement-kit.github.io/>.
+// Measurement-kit is free software. See AUTHORS and LICENSE for more
+// information on the copying conditions.
+#ifndef MEASUREMENT_KIT_NEUBOT_DASH_HPP
+#define MEASUREMENT_KIT_NEUBOT_DASH_HPP
+
+#include <functional>
```

Comment:
Strictly speaking, header not needed, because already pulled in by `common.hpp`.